spec/Order_Repo_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'Order_Repo'
+require 'Order'
+
+def reset_shop_table
+    seed_sql = File.read('spec/seeds_test.sql')
+    connection = PG.connect({ host: '127.0.0.1', dbname: 'shop_manager_test' })
+    connection.exec(seed_sql)
+  end
+  
+ RSpec.describe Orders_Repo do
+    before(:each) do 
+      reset_shop_table
+    end
```

Comment:
`after(:each) do
     reset_shop_table
end`
This truncates the tables after each test. Doesn't impact your current code but good to know